apps/src/codebridge/styles/codebridge.scss

```diff
@@ -14,3 +14,23 @@
   fill: #eee;
 }
 
+// Style the counter used by neighborhood
```

Comment:
You mentioned that 'The styles for Java Lab/other mazes were in `common.scss`, which doesn't seem to be used in lab2.' Just curious whether we would want to include styles from `common.scss` in `lab2` as we did for blockly? https://github.com/code-dot-org/code-dot-org/blob/999ed00d6ad0f9d28c70fef5cb22a788311739a9/apps/style/lab2/style.scss#L1

It would be nice to have styles that apply to both legacy and lab available somewhere but `common.scss` seems to apply to legacy labs only.

Probably makes sense to include here then, but maybe add a note that this style is also defined for legacy labs in `common.scss` in case it's updated in the future?

Reply:
Added comment!